### ISSUE TITLE
MatrixMate problems in multi-site setup

### ISSUE THREAD

**Issue by ragnarfrosti** [vaersaagod/matrixmate#17]
We are having some issues using MatrixMate in a multi-site setup when editing an entry.

You probably need to include the siteId parameter in `getEntryById` function (row 130ish) in `onAfterLoadPlugins` function in MatrixMate.php, else it will not find any entries from other sites than the default site.

```
if ($entryId && $entry = Craft::$app->getEntries()->getEntryById($entryId)) {
    $entryType = $entry->getType();
}
```
So it will look something like this:

```
$siteHandle = $request->getParam('site');
$siteId = 1;
if ($siteHandle) {
    $site = Craft::$app->getSites()->getSiteByHandle($siteHandle);
    $siteId = $site->id;
}

if ($entryId && $entry = Craft::$app->getEntries()->getEntryById($entryId, $siteId)) {
    $entryType = $entry->getType();
}
```

Best regards,
Ragnar

**Comment by mmikkel** [comment]
Hi Ragnar,

Good catch, thanks. This should only be an issue for non-propagating sections in multi-site setups, but I'll definitely get a fix out ASAP :)

**Comment by mmikkel** [comment]
I just published [MatrixMate 1.2.5](https://github.com/vaersaagod/matrixmate/releases/tag/v1.2.5), which should resolve this issue.
